fetch/wire_types.go

```diff
@@ -52,6 +52,12 @@ type MeshHashes struct {
 	Hashes []types.Hash32
 }
 
+type EpochData struct {
+	Beacon types.Beacon
+	Weight uint64
```

Comment:
can we please drop this two fields? both of them should have zero impact on downloading atxs. if they do then something is clearly wrong

Reply:
i ended up dropping beacon and kept weight for logging the amount of ignored weight from the missing atxs

Reply:
but why to send it? it is not like this number can be trusted